go/vt/vtgate/vstream_manager_test.go

```diff
@@ -274,6 +274,74 @@ func TestVStreamChunks(t *testing.T) {
 	assert.Equal(t, int32(100), ddlCount.Get())
 }
 
+func TestVStreamManagerGetCells(t *testing.T) {
+	type testArgs struct {
+		name        string
+		optCells    string
+		cellAlias   bool
+		resultCells []string
+	}
+
+	tcases := []*testArgs{
+		{"default-local", "", false, []string{"aa"}},
+		{"default-local-cell-alias", "", true, []string{"local:aa", "region1"}},
+		{"with-opt-cells", "local:,bb,cc", true, []string{"local:aa", "bb", "cc"}},
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	cell := "aa"
```

Comment:
We should make the local cell a test case variable too. That would have likely highlighted the "lost cell" issue too.